base/namedtuple.jl

```diff
@@ -89,6 +89,7 @@ firstindex(t::NamedTuple) = 1
 lastindex(t::NamedTuple) = nfields(t)
 getindex(t::NamedTuple, i::Int) = getfield(t, i)
 getindex(t::NamedTuple, i::Symbol) = getfield(t, i)
+getindex(t::NamedTuple, r::AbstractUnitRange{T}) where T = ([getfield(t, ri) for ri in r]...,)
```

Comment:
perhaps a better way to do this is:
```suggestion
getindex(t::NamedTuple, r::AbstractVector) = ((getfield(t, ri) for ri in r)...,)
```